docs/src/cookbooks/verilog-vs-chisel.md

```diff
@@ -0,0 +1,830 @@
+<!Doctype html>
+<html>
+
+# Verilog vs Chisel Side-By-Side
+
```

Comment:
Do we want to add a brief intro to this page? something like:

"This page serves as a quick introduction to Chisel for those familiar with Verilog. It is by no means a comprehensive guide of everything Chisel can do. Feel free to [file an issue](https://github.com/chipsalliance/chisel3/issues) with suggestions of things you'd like to see added to this page. "